src/validation_workflow/docker/spinup_docker_container.py

```diff
@@ -0,0 +1,62 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import shutil
+import subprocess
+from subprocess import PIPE
+from typing import Any
+
+from system.temporary_directory import TemporaryDirectory
+
+"""
+This class is to spin up docker containers with the OS and OSD images specificed earlier.
+It uses docker-compose to bring up docker containers.
+It returns True if containers are Up.
+"""
+
+
+class RunDocker:
```

Comment:
I don't think we need a class to spin up the docker containers right?
Why not put all of this under `install` and `start_cluster` here https://github.com/opensearch-project/opensearch-build/pull/3038/files#diff-968cdbdd15c8ab34a3bc9aa41df188abc7e459e0fc56f6affc36be763b78b898R48-R54

Reply:
Thanks, @gaiksaya .. Because there are many lines of code to spin up the docker containers, and I am afraid the validation_docker.py will become too many lines of code; hence, I have it seperated from the validation_docker.py. And, sure, I can place it as a method in the validation_docker.py as well.

Reply:
That's okay I think! Also would suggest to use better naming convention for variables. OS1_name  does not give me much information on what that is. Use complete name if possible.